app/helpers/application_helper.rb

```diff
@@ -1648,12 +1648,8 @@ def listicon_image_tag(db, row)
       vendor = host ? host.vmm_vendor_display.downcase : "unknown"
       image = "svg/vendor-#{vendor}.svg"
     elsif db == "MiqAction"
-      action = @targets_hash[@id.to_i]
-      image = if action && action.action_type != "default"
-                "100/miq_action_#{action.action_type}.png"
-              else
-                "100/miq_action.png"
-              end
+      action = @targets_hash[row['id']]
```

Comment:
Aah, so it was broken before, ok :) .. but in that case, please do fix all of them - Host has the same problem in that function.

And there's no point in mixing `row['id']` and `from_cid(@id)` if they're supposed to be the same thing .. pick one ;)